src/cpl/nuopc/atm_import_export.F90

```diff
@@ -807,16 +838,18 @@ subroutine export_fields( gcomp, cam_out, rc)
     !-------------------------------
 
     ! input/output variables
-    type(ESMF_GridComp)           :: gcomp
-    type(cam_out_t) , intent(in)  :: cam_out
-    integer         , intent(out) :: rc
+    type(ESMF_GridComp)              :: gcomp
+    type(ESMF_Mesh) , intent(in)     :: model_mesh
+    type(ESMF_Clock), intent(in)     :: model_clock
```

Comment:
I don't believe this is used.

Reply:
It looks like both `model_mesh` and `model_clock` are both used in the call to `stream_ndep_init`, so I think they are actually used?  Of course please let me know if I mis-understood your suggestion!

Reply:
Oops - my bad! I meant to put this comment on line ~849:

`type(ESMF_Clock)  :: clock`

That's what I don't think is used!

Reply:
Should be removed now!